samples/nrf9160/modem_shell/src/main.c

```diff
@@ -143,7 +142,6 @@ void main(void)
 	}
 	printk("Initialized modemlib\n");
 
-	at_cmd_init();
```

Comment:
I guess we cannot remove this before all dependencies have been converted to use `nrf_modem_at.h` and `at_monitor.h`?
At least SMS and modem_info libraries haven't been changed and SMS doesn't work with this modem_shell branch.

PR for SMS: https://github.com/nrfconnect/sdk-nrf/pull/6314

Reply:
Yep, there's the SMS and `modem_info` which I am porting right now. Are there more dependencies?

Reply:
Based on what I could find from nrf, `lwm2m` is likely the only additional one.
I guess lwm2m has dependency to SMS, which will take some time due to the linking bug in Zephyr (which prevents Unity tests from passing):
https://github.com/nrfconnect/sdk-nrf/pull/6314#discussion_r765726406

Reply:
I added DNM label so that this won't be merged before we get the dependencies in place.

Reply:
@trantanen all the dependencies have been ported to the new AT APIs, I'll dismiss your review to get this merged.